standard_schema/prerelease/HED8.3.0.mediawiki

```diff
@@ -243,6 +238,18 @@ Each term in this vocabulary has a human-readable description and may include ad
 ***** Shoulder <nowiki>{hedId=HED_0012227} [Joint attaching upper arm to trunk.]</nowiki>
 ***** Upper-arm <nowiki>{hedId=HED_0012228} [Portion of arm between shoulder and elbow.]</nowiki>
 ***** Wrist <nowiki>{hedId=HED_0012229} [A joint between the distal end of the radius and the proximal row of carpal bones.]</nowiki>
+**** Lower-extremity <nowiki>{hedId=HED_0012195} [Refers to the whole inferior limb (leg and/or foot).]</nowiki>
+***** Ankle <nowiki>{hedId=HED_0012196} [A gliding joint between the distal ends of the tibia and fibula and the proximal end of the talus.]</nowiki>
+***** Calf <nowiki>{hedId=HED_0012197} [The fleshy part at the back of the leg below the knee.]</nowiki>
+***** Foot <nowiki>{hedId=HED_0012198} [The structure found below the ankle joint required for locomotion.]</nowiki>
+****** Big-toe <nowiki>{hedId=HED_0012199} [The largest toe on the inner side of the foot.]</nowiki>
+****** Heel <nowiki>{hedId=HED_0012200} [The back of the foot below the ankle.]</nowiki>
+****** Instep <nowiki>{hedId=HED_0012201} [The part of the foot between the ball and the heel on the inner side.]</nowiki>
+****** Little-toe <nowiki>{hedId=HED_0012202} [The smallest toe located on the outer side of the foot.]</nowiki>
+****** Toes <nowiki>{hedId=HED_0012203} [The terminal digits of the foot.]</nowiki>

```

Comment:
do we want this to be plural or singular.

Reply:
That is a very good question. Since Toes already have a HED ID, I did not want to remove it. However, to comply with HED nomenclautre, it seems that singular form is more preferred. Finger is also singular. However, Teeth is also there.

Reply:
A finger as singular makes sense since a finger pushes a button etc.   I don't know if toes are used individually.  One might grind ones teeth, I don't know if something would pertain to a particular tooth.

Reply:
After discussion in HWG, it seems that both singular and plural forms are very useful. So we will keep both.